### DISCUSSION TITLE
how to use on IOS?

### DISCUSSION THREAD

**Discussion by Morgandri1** [beeware/briefcase#830] [Q&A]
how can i run my beeware app on my iphone? 

the tutorial app crashes on startup.

**Comment by freakboy3742** [comment]
We're going to need a little more detail than "it crashes". Can you provide specifics on:
* your macOS version
* your iOS version (either the version on your phone, or the simulator version that you're using
* Your Xcode version
* The version of Python you're using
* Any Briefcase log files that have been produced. If no Briefcase log files have been produced by the crash, please run `briefcase run iOS --log` to force generation of a log file.

**Reply by Morgandri1** [reply]
right sorry.

im on ventura 13 beta 6 (though this has failed on catalina 10.13, big sur 11, and Monterey 12 as well)
ios 15.6 (this has also failed on 14.6 and 16 beta 4)
xcode beta 14 (this has also failed on 11.4, and 13.6 i believe)
python 3.10.4 (has also failed on 3.10.2 and 3.9.8)

i would like to reiterate what i mean by crashing.
its not really a crash, but the xcode objective-c stuff has lots of errors.

it only shows a blank white screen, no labels or anything. it shows the "made with beeware" screen, then plain white
this happens on the IOS simulator as well as my personal phone

**Reply by mhsmith** [reply]
Please post the the log file as mentioned above. And if you see any error messages which aren't in the log file, please post them too.

**Reply by Morgandri1** [reply]
sorry, i tried to replicate the errors again and no issues. sorry to waste time
